crypto/x509/x509_v3.c

```diff
@@ -118,15 +120,13 @@ STACK_OF(X509_EXTENSION) *X509v3_add_ext(STACK_OF(X509_EXTENSION) **x,
         loc = n;
 
     if ((new_ex = X509_EXTENSION_dup(ex)) == NULL)
-        goto err2;
+        goto err;
     if (!sk_X509_EXTENSION_insert(sk, new_ex, loc))
         goto err;
```

Comment:
These look like they should have an ERR_R_X509_LIB raised.

Reply:
Yup, except `ERR_R_CRYPTO_LIB`, because that's the "sub-system" that all `sk_` functions belong in.